docs/pages/access-controls/reference.mdx

```diff
@@ -260,6 +262,10 @@ spec:
 
 ### Extended Node labels syntax
 
+Any string starting with `^` and ending with `$` will be interpreted as a regex and not as a literal string.
```

Comment:
```suggestion
Any string starting with `^` or ending with `$` will be interpreted as a regex and not as a literal string.
```

It's technically not an and, either one of these conditions alone is enough for us to treat it as a regex.